src/app.js

```diff
@@ -1 +1,34 @@
 'use strict';
+
+const fs = require('fs');
+
+const path = require('path');
+
+const copyFile = (src, dest) => {
+  const srcPath = path.join(__dirname, src);
+  const destPath = path.join(__dirname, dest);
+
+  if (srcPath === destPath) {
+    return;
+  }
+
+  try {
+    fs.copyFileSync(srcPath, destPath);
+  } catch (err) {
+    throw new Error(err);
```

Comment:
Instead of rethrowing the error, you can print an error message and exit the process with a non-zero status code